rfcs/054-id-resolver/README.md

```diff
@@ -0,0 +1,155 @@
+so a new stage could populate these properly.
+
+### Validation
+
+Currently, there is no validation as to whether the given identifier actually exists within the given scheme.
+Even if it does, there is no validation as to whether there is any correspondence between the name of the concept
+and the identifier.
+
+As a result, there are some errors that can currently only be detected by eye.
+
+* Some MeSH identifiers have been found wrongly assigned to the LCSH scheme.  
+* Two LCNames identifiers (n50034502 and n2001003970) do not exist
+* Some identifiers in the LCSH scheme have been parsed/written incorrectly, so they are not identifiers at all.
+* The wrong field was being extracted from some TEI records to find the LCSH id.
+
+In addition, there may be some that are very difficult to detect by eye.  A real identifier that refers to something different
+to the label of the field would only be detected when someone follows a link and finds themselves on a wholly unexpected Concept page.
+`n2001003970`, mentioned above, does not exist but `nr2001003970` does.  
+However, that identifer refers to the "Partnership for Governance Reform in Indonesia", whereas
+the associated label in the offending Work is "Pseudo Aristotele., De coloribus., Latin".
+
+## What about the Concepts Embedder?
+
+The Concepts Embedder is a proposed new part of the catalogue pipeline that uses Concepts in the Concept Store.
+It will include information from the Knowledge Graph.  The information described above would be available at the Concept Embedder
+stage, but it is too late in the Catalogue Pipeline.
+
+There is also a circular problem.  If a Concept is not used in the Catalogue, then it does not enter the Knowledge Graph and 
+therefore does not become available at the Concept Embedder stage.  This may mean extra complexity at that stage
+to determine the sameness of Concepts that have been recorded with the wrong Authority Namespace or Type.  Or where
+an Authoritative Concept has been recorded as a top-level, atomic concept in one Work, and as part of a compound in another.
+By resolving those issues before the ID Minter, those Concepts will actually be the same Concept when they reach the
+Concepts Pipeline, rather than multiple Concepts that need to be understood as synonymous.
+
+The proposed stage here operates on Authoritative Concept Identifiers, ensuring that the resulting Catalogue Concepts 
+are correct and consistent before new Canonical Identifiers are minted for them.
+
+## How?
+
+### Concepts Pipeline/API
+The Concepts Pipeline Ingestor(s) should, where available, store the official types (maybe translated to our types) 
+and compositions of each Concept in the authoritative-concepts index.
+
+The Concepts API should have a service that can be called with one or more authoritative identifiers, and returns that information
+from the authoritative index.  This may be a separate private/internal API instance.
```

Comment:
I think we could probably make the normal Concepts API do this, right?

Reply:
I think that would be simplest.